default/AI/ProductionAI.py

```diff
@@ -9,25 +9,28 @@
 import PlanetUtilsAI
 import PriorityAI
 import ColonisationAI
-import EnumsAI
 import MilitaryAI
 import ShipDesignAI
 import time
 import cProfile, pstats, StringIO
+
+from EnumsAI import PriorityType, EmpireProductionTypes, MissionType, get_priority_production_types, FocusType, \
+    ShipRoleType
+from EnumsAI import ShipDesignTypes
```

Comment:
Oops.  Pycharm add import with `quick-fix` in that way. Parentheses support was added about 10 years ago in Python 2.4.